src/protocols/ethereum/libs/EthereumAdapter.ts

```diff
@@ -245,6 +246,53 @@ export class EthereumAdapter extends BaseProtocolAdapter {
     }
   }
 
+  override async transferPreparedTransaction(transactionData: any): Promise<ITransferResponse> {
```

Comment:
Maybe you can destructure the argument properties so there would be less code in the function's body?